packages/v4/src/content/design-guidelines/demos/card-view/card-view.md

```diff
@@ -0,0 +1,81 @@
+---
+id: Card view
+section: demos
+---
+
+
+A **card view** is a grid of cards in a gallery to facilitate browsing. Card views are typically used to present a summary of some data set where the user can drill-down from any card to see detailed content.
+
+<img src="./img/card-view.png" alt="card view" />
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+
+### Background colors for card views
+Use a white or a gray background for your card views.
+
+Use a white background if your content view can change. Examples:
+- A card that opens an expandable panel on the right
+- A card view that toggles into a table or list view
+
```

Comment:
```suggestion
**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow.
```

Bumping this note to above the example image, that way it won't get lost between the image and the next background section.